tests/models/glm4v_moe/test_modeling_glm4v_moe.py

```diff
@@ -310,7 +311,8 @@ def get_model(cls):
 
     @classmethod
     def tearDownClass(cls):
-        del cls.model
+        if hasattr(cls, "model"):
+            del cls.model
```

Comment:
it's quite strange the issue only happens on CircleCI , and in this case, only when a non-integration tests is run

For example

> python3 -m pytest -v tests/models/glm4v_moe/test_modeling_glm4v_moe.py -k "Glm4vMoeIntegrationTest"

this doesn't have error

but 

> python3 -m pytest -v tests/models/glm4v_moe/test_modeling_glm4v_moe.py -k "Glm4vMoeIntegrationTest or test_attention_outputs"

have error

On daily CI runners, the second one has no error neither.

Not sure what is going on with CircleCI environment, but yeah let's keep this.